.circleci/config.yml

```diff
@@ -75,9 +75,11 @@ jobs:
       - run:
           name: "Check out sample project."
           command: git clone https://github.com/CircleCI-Public/circleci-demo-go.git ~/project
-      - go/load-cache
+      - go/load-cache:
+          key: "integration"
```

Comment:
For the cache keys, I'd add the checksum for "go.sum".

So it would end up like this:

```
key: integration-{{ checksum "go.sum" }}
```

Reply:
@felicianotech The way the command is already designed, it is hard coded to be based off of "go.mod"

`        - << parameters.key >>-{{ checksum "go.mod"  }}`
https://github.com/CircleCI-Public/go-orb/blob/c30099d598260492d859ba9400b6df2b58588a1d/src/commands/load-cache.yml#L10

Reply:
Ah good catch. I need to change that file but that's a separate PR. The other problem here is assuming that go.mod/go.sum would be located in the root directory. Again, another PR though. 

My mistake, this is good then.